File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int x;
    do
    {
        x = get_int("Please enter height!: ");
    }
    while (x < 1 || x > 8);

    for (int a = 0; a < x; a++)
    {
        for (int b = 0; b < x; b++)
        {
            if (a + b < x - 1)
                printf(" ");
            else
                printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

while True:
    n = get_int('Please enter height!: ')
    if n > 0 and n < 9:
        break

for x in range(0, n , 1):
    for y in range(0, n , 1):
        if (x + y < n - 1):
            print(" ", ends="")
        else:
            print('#', end="")
    print()
